site-permissions/site-permissions-impl/src/main/java/com/duckduckgo/site/permissions/impl/SitePermissionsRepository.kt

```diff
@@ -105,6 +122,10 @@ class SitePermissionsRepositoryImpl @Inject constructor(
                 val isMicAlwaysAllowed = sitePermissionForDomain?.askMicSetting == SitePermissionAskSettingType.ALLOW_ALWAYS.name
                 permissionGrantedWithin24h || isMicAlwaysAllowed
             }
+            PermissionRequest.RESOURCE_PROTECTED_MEDIA_ID -> {
+                val isDrmAlwaysAllowed = sitePermissionForDomain?.askDrmSetting == SitePermissionAskSettingType.ALLOW_ALWAYS.name
+                permissionGrantedWithin24h || isDrmAlwaysAllowed
```

Comment:
If we are doing the sessions thing, do you need permissionGrantedWithin24h here?